### DISCUSSION TITLE
A Freestanding Rust Binary (ko)

### DISCUSSION THREAD

**Discussion by giscus** [phil-opp/blog_os#1255] [Post Comments (translated)]
# A Freestanding Rust Binary (ko)

운영체제 커널을 만드는 첫 단계는 표준 라이브러리(standard library)를 링크하지 않는 Rust 실행파일을 만드는 것입니다. 이 실행파일은 운영체제가 없는 bare metal 시스템에서 동작할 수 있습니다.

https://os.phil-opp.com/ko/freestanding-rust-binary/

**Comment by ing03201** [comment]
재밌게 공부하고있습니다
좋은 컨텐츠 감사합니다

**Comment by dongsub-joung** [comment]
Thx.  

It's beneficial for me.

**Comment by Takikocat** [comment]
덕분에 훌륭한 계획이 생긴 것 같아요! 감사합니다. Thx.

**Reply by Takikocat** [reply]
Thanks to that, I think I have a great plan now!
